### PR TITLE
tests: first core20 test fixes

### PR DESCRIPTION
**Pull request by mvo5**
Initial set of fixes from Sergios branch that runs all of the main tests.

### CODE REVIEW

File: tests/main/install-sideload/task.yaml [hunk 1]
```diff
@@ -71,7 +71,15 @@ execute: |
...
     echo "All snap blobs are 0600"
-    test "$( find /var/lib/snapd/{snaps,cache,seed/snaps}/ -type f -printf '%#m\n' | sort -u | xargs )" = "0600"
+    test "$( find /var/lib/snapd/{snaps,cache}/ -type f -printf '%#m\n' | sort -u | xargs )" = "0600"
+
+    # shellcheck source=tests/lib/systems.sh
+    . "$TESTSLIB/systems.sh"
+    # The "seed/" dir is on a FAT partition on uc20 so the permissions are
+    # different here.
+    if ! is_core20_system; then
+        test "$( find /var/lib/snapd/seed/snaps/ -type f -printf '%#m\n' | sort -u | xargs )" = "0600"
```
Comment (sergiocazzolato) [inline tests/main/install-sideload/task.yaml]: I think this is not needed due to this test is not executed on ubuntu core

File: tests/main/install-store/task.yaml [hunk 2]
```diff
@@ -44,4 +44,12 @@ execute: |
     snap install --edge test-snapd-complexion
 
     echo "All snap blobs are 0600"
-    test "$( find /var/lib/snapd/{snaps,cache,seed/snaps}/ -type f -printf '%#m\n' | sort -u | xargs )" = "0600"
+    test "$( find /var/lib/snapd/{snaps,cache}/ -type f -printf '%#m\n' | sort -u | xargs )" = "0600"
```
Comment (sergiocazzolato) [inline tests/main/install-store/task.yaml]: This is not enough for core 20, some snaps on /var/lib/snapd/snaps/ also have permissions 0700

google:ubuntu-core-20-64 .../tests/main/install-sideload# find /var/lib/snapd/snaps/ /var/lib/snapd/cache/  -type f -printf '%p %#m\n'
/var/lib/snapd/snaps/test-snapd-sh_2.snap.partial 0600
/var/lib/snapd/snaps/core20_284.snap 0700
/var/lib/snapd/snaps/core_8268.snap 0600
/var/lib/snapd/snaps/test-snapd-devmode_x1.snap 0600
/var/lib/snapd/snaps/basic_x1.snap 0600
/var/lib/snapd/snaps/test-snapd-tools_6.snap.partial 0600
/var/lib/snapd/snaps/pc-kernel_362.snap 0700
/var/lib/snapd/snaps/test-snapd-tools_x1.snap 0600
/var/lib/snapd/snaps/test-snapd-devmode_x2.snap 0600
/var/lib/snapd/snaps/basic-desktop_x1.snap 0600
/var/lib/snapd/snaps/pc_81.snap 0700
/var/lib/snapd/snaps/jq_6.snap.partial 0600
/var/lib/snapd/snaps/test-snapd-tools_x2.snap 0600
/var/lib/snapd/snaps/snapd_x1.snap 0700
/var/lib/snapd/snaps/test-snapd-rsync-core20_1.snap 0600
/var/lib/snapd/cache/7946b4de33a1b621a7c9f22a1463d858c55d09db7294b187527127ef1c71b126dc7d687f294d89f7c2eb63a644af982f 0600
/var/lib/snapd/cache/9f0ee5d3dd59ae4e46b04ab358778729a4b29992177cecf6511c070c0e67f67540e557f6a7c48e985e658c93c4bbb2f1 0600
/var/lib/snapd/cache/b25a38e0a01970659c840c675788f2143a274f7b08a99f3ee765831d9c19b502c1af1cefb0caced3105a07f0bb2cc6ab 0600

Comment (sergiocazzolato) [reply tests/main/install-store/task.yaml]: @mvo5, is this expected or needs to be fixed?

Comment (sergiocazzolato) [reply tests/main/install-store/task.yaml]: we should revert this file, now is failing the test on ubuntu 14.04: https://paste.ubuntu.com/p/WjSHrW2bTv/

File: tests/main/install-sideload/task.yaml (hunk 1 above)
Comment (sergiocazzolato) [reply tests/main/install-sideload/task.yaml]: The test now is failing in all the ubuntus: https://paste.ubuntu.com/p/tqQDfd7mNM/

Comment (mvo5) [reply tests/main/install-sideload/task.yaml]: Thank you!

**Pull request closed without merging**